user/models.py

```diff
@@ -1,12 +1,12 @@
 from django.db import models
 
 class User(models.Model) :
-    identity    = models.CharField(max_length=20, unique=True, null=False)
-    password    = models.CharField(max_length=200, null=False)
-    phone       = models.CharField(max_length=15, unique=True, null=False)
-    name        = models.CharField(max_length=20, null=False)
-    email       = models.CharField(max_length=45, unique=True, null=True)
-    address     = models.CharField(max_length=100, null=True)
+    identity = models.CharField(max_length=45, unique=True, null=False)
+    password = models.CharField(max_length=45, null=False)
+    phone    = models.CharField(max_length=11, unique=True, null=False)
+    name     = models.CharField(max_length=45, null=False)
+    email    = models.CharField(max_length=45, unique=True, null=False)
+    address  = models.TextField(null=False)
```

Comment:
address로 TextField는 너무 크다는 생각이 드네요.. 
`CharField`와 `TextField` 차이를 확인해보시면 도움이 될 것 같습니다~

```suggestion
    address  = models.CharField(..
```